CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Remove AttributeMessageType (#4020)
 - Remove `mem-ballast-size-mib`, already deprecated and no-op (#4005).
 - Remove `AttributeHTTPStatusText` const, replaced with `"http.status_text"` (#4015, contrib/#5182).
+- Remove squash on `configtls.TLSClientSetting` in `confighttp` and `cnofiggrpc` config (#4063).
```

Comment:
Please explain in more user impact than programmatically what changed. Configs for tls are moved one level deeper ...

Reply:
Updated. Thanks